# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/base/moz.build
# Commit: 92148b810f22
# Full Hash: 92148b810f228e4e9e84255682fe888a5bee07d5
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2023-12-24 09:09:23
# Regressor Bug: 779684
# File Overlap Count: 1
# Description:
#   Bug 779684 - part 2: Get rid of `nsFrameTraversal` and `nsIFrameTraversal` r=emilio
#   
#   The class is used only for creating `nsFrameIterator`, but it's unnecessary
#   so that we can get rid of it and its interface.
#   
# ==============================================================================

diff -r d4411b38cae7 -r 92148b810f22 layout/base/moz.build
--- a/layout/base/moz.build	Sun Dec 24 06:01:19 2023 +0000
+++ b/layout/base/moz.build	Sun Dec 24 06:01:19 2023 +0000
@@ -48,7 +48,6 @@
     "nsFrameManager.h",
     "nsFrameTraversal.h",
     "nsGenConList.h",
-    "nsIFrameTraversal.h",
     "nsIPercentBSizeObserver.h",
     "nsIReflowCallback.h",
     "nsLayoutUtils.h",